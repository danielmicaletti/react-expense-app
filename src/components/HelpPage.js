import React from 'react';

const HelpPage = () => (
	<div>
		Help Page Component FILE
	</div>
);

export default HelpPage;