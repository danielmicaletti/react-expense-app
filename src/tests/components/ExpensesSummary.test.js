import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('should render correct ExpensesSummary on single expense', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={245} />);
	expect(wrapper).toMatchSnapshot();
});

test('should render correct ExpensesSummary on multiple expenses', () => {
	const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={2452332} />);
	expect(wrapper).toMatchSnapshot();
});