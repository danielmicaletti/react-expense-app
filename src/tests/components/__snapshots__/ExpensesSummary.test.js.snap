// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correct ExpensesSummary on multiple expenses 1`] = `
<div>
  <h3>
    Viewing 
    23
     
    expenses
     totalling 
    $24,523.32
  </h3>
</div>
`;

exports[`should render correct ExpensesSummary on single expense 1`] = `
<div>
  <h3>
    Viewing 
    1
     
    expense
     totalling 
    $2.45
  </h3>
</div>
`;
